sites/dwhdelft.nl/pages/jongenout.vue

```diff
@@ -20,14 +20,30 @@ nl:
   join:
     button: Meld je nu aan
   team:
-    - name: Hidde
-      pronouns: hij/hem
-      description:
-        - 'Hoi, ik ben Hidde, 24 jaar en ik loop al een tijdje rond bij DWH/Outsite. Met mijn ervaring als middelbare school docent hoop ik ook een bijdrage te kunnen leveren bij J&O Delft.'
     - name: Dante
-      pronouns: die/diens
+      pronouns: die/hun
       description:
         - 'Heeyhey, dit is Dante. Ik ben Jong&Out gaan doen omdat een soort gelijke groep mij op de middelbare heel veel plezier gebracht heeft. Deze ervaring wil ik graag doorgeven en er iets gaafs van maken.'
+    - name: Thomas
+      pronouns: hij/hem
+      description:
+        - 'Hoi, ik ben Thomas! Als begeleider bij Jong & Out Delft vind ik het geweldig om bij te dragen aan een veilige ruimte waar queer jongeren zichzelf kunnen zijn, zeker omdat ik dat niet op mijn middelbare school had!'
+    - name: Dim
+      pronouns: hij/hem
+      description:
+        - 'Hiii! Ik ben Dim! Ik kwam zelf vroeger bij Jong & Out en heb hier vrienden voor het leven gemaakt! Zulke vrienden gun ik iedereen en daarom ben ik zelf begeleider bij Jong & Out geworden!'
+    - name: Meander
+      pronouns: die/hun
+      description:
+        - 'Hey, ik ben Meander! Zelf heb ik ook mijn J&O tijd gehad, het was zo’n fijne en vertrouwde omgeving, dat wil ik daarom ook creëren voor de volgende generatie aan J&O’ers 😝!!'
```

Comment:
```suggestion
        - 'Hey, ik ben Meander! Zelf heb ik ook mijn J&O tijd gehad, het was zo’n fijne en vertrouwde omgeving, dat wil ik daarom ook creëren voor de volgende generatie J&O’ers 😝!!'
```
dan past het net wat lekkerder op drie regels :)